src/policy.rs

```diff
@@ -33,6 +33,11 @@ pub struct Policy {
     /// List of ContextAwareResource the policy is granted access to.
     /// Currently, this is relevant only for waPC based policies
     pub ctx_aware_resources_allow_list: HashSet<ContextAwareResource>,
+
+    /// Minimal Kubewarden stack version required to run the policy.
+    /// This is relevant if a policy requires a host capability available after
+    /// some specific Kubewarden version.
+    pub minimal_kubewarden_version: Option<String>,
```

Comment:
I would instead use a [`Version`](https://docs.rs/semver/1.0.17/semver/struct.Version.html) object to store that. I've seen this crate has also `serde` support, I think this can be used to nicely serialize/deserialize the string into this object.

The main advantage of doing that would be to immediately spot wrong input provided by the user